### ISSUE TITLE
Will this be updated with the new changes

### ISSUE THREAD

**Issue by Muzik4Machines** [dphuang2/PoGoBag#99]
because right now it tells me my gyarados (level 30 yesterday) is 40.5 (and i'm 34)

**Comment by b1ggjoe** [comment]
Yep, Niantic did a system wide CP adjustment to everyone's Pokemon.
Traditional OP Pokemon got their CP lowered. Others had their CP raised.

J.

**Comment by Muzik4Machines** [comment]
i know, my arcanines are f'd lol but i meant will this script be updated with the new values or is it abandonned?

**Comment by b1ggjoe** [comment]
Hmmm...I think if we login again manually, then it should update
accordingly?

**Comment by Muzik4Machines** [comment]
no, i did, i'm talking about the home run version, since the pogobag.me site seems gone, the IV are good but the level it shows are off anyway http://imgur.com/a/qVH9o

**Comment by SonGokuSSJ** [comment]
I noticed that also yesterday. But the alternatives (PokemonGoReader and pokemon-go-stats) showed the correct CP for me. I did not take a look at the level though as pokemon-go-stats does not show the level and attacks. I First thought it was a "you got too many Pokemon"-bug because when I First launched it I was near 930 Pokemon and when I was finished will all evolutions and transfers and refreshed the data by logging in again (I am also using the selfhosted version of PoGoBag to show my pokemon around) the 3 to 5 Pokemon I had with wrong value also showed the correct value again. 

For clarification: With correct CP value and lower CP value I mean in comparision to ingame, i.e. I have a Pidgeot with 79 CP (ingame). It has shown 73CP in PoGoBag, but both PokemonGoReader and pokemon-go-stats were showing 79CP. I noticed this with 2 to 4 other Pokemon.
